scripts/Units/armfido.bos

```diff
@@ -6,13 +6,14 @@
 piece  barrel, turret, loturret, flare, sleeve, torso, pelvis, lthigh, lleg, lfoot, rthigh, rleg, rfoot, luparm, lloarm, lhand, ruparm, rloarm, rhand,
        eyes, shoulder, ltooth, rtooth, aimy1, aimx1;
 
-static-var  restore_delay, Static_Var_2, gun_2, statechg_DesiredState,
-			statechg_StateChanging, moveSpeed, currentSpeed, wpn1_lasthead, wpn2_lasthead, animSpeed, maxSpeed, animFramesPerKeyframe, bMoving;
+static-var  restore_delay, Static_Var_2, moveSpeed, currentSpeed, animSpeed, maxSpeed, animFramesPerKeyframe, bMoving;
```

Comment:
`Static_Var_2` -> `currentWeapon`, and add constants for the weapons, e.g. `#define GAUSS 0`, then the functionality is clear in the code, and explanatory comments aren't needed.
`if (currentWeapon == GAUSS) { }`

Reply:
That makes sense, will do